src/nvim/eval.lua

```diff
@@ -4470,11 +4470,12 @@ M.funcs = {
 
       Examples: >vim
       	echo getscriptinfo({'name': 'myscript'})
-      	echo getscriptinfo({'sid': 15}).variables
+      	echo getscriptinfo({'sid': 15})[1].variables
```

Comment:
```suggestion
      	echo getscriptinfo({'sid': 15})[0].variables
```

Reply:
uh duh Lua's 1-based indexing is getting into me.